arwen/context/errors.go

```diff
@@ -11,3 +11,5 @@ var ErrFunctionRunError = errors.New("function run error")
 var ErrFuncNotFound = errors.New("function not found")
 
 var ErrReturnCodeNotOk = errors.New("return not is not ok")
+
+var ErrNotEnoughGas = errors.New("not enough gas")
```

Comment:
Is this needed? We have `vmcommon.OutOfGas`.

Reply:
no. as that is an error code only. here it is the full error.